cli/slsa-verifier/verify/options.go

```diff
@@ -45,31 +45,41 @@ type VerifyOptions struct {
 
 var _ Interface = (*VerifyOptions)(nil)
 
+var CommonFilenameExtensions = []string{"sigstore", "intoto", "intoto.jsonl", "json"}
```

Comment:
```suggestion
var CommonAttestationFilenameExtensions = []string{"sigstore", "intoto", "intoto.jsonl", "json"}
```

Reply:
This is also being used for provenance filenames per earlier comments. Do we still want to rename it as suggested?

Reply:
Yes, provenance and attestation are mostly interchangeable terms, but attestation is more generic.

Reply:
Done.